Assets/SEE/UI/Window/MetricWindow.cs

```diff
@@ -0,0 +1,215 @@
+            foreach (GameObject obj in gameObjects)
+            {
+                attributesList.Add(AttributeName(obj));
+            }
+            IEnumerable<(int score, string attribute)> searchResults
+                = Process.ExtractTop(query, attributesList, limit: limit, cutoff: cutoff).Select(x => (x.Score, x.Value));
+            searchResults = searchResults.OrderByDescending(x => x.score);
+
+            return searchResults.Select(x => x.attribute);
+        }
+
+        /// <summary>
+        /// Returns the name of a node attribute stored in the first child of the <paramref name="metricRow"/>.
+        /// Parameter <paramref name="metricRow"/> is assumed to represent a row in the metric window providing
+        /// the name and value of a node attribute (metric).
+        /// </summary>
+        /// <param name="metricRow">a game object representing a pair of an attribute name and an attribute value</param>
+        /// <returns>name of the node attribute</returns>
+        private static string AttributeName(GameObject metricRow)
+        {
+            return Attribute(metricRow).text;
+        }
+
+        /// <summary>
+        /// Returns the TMP container of <paramref name="metricRow"/> holding the attribute name.
+        /// Parameter <paramref name="metricRow"/> is assumed to represent a row in the metric window providing
+        /// the name and value of a node attribute (metric).
+        /// </summary>
+        /// <param name="metricRow">a game object representing a pair of an attribute name and an attribute value</param>
+        /// <returns>the TMP holding the attribute name</returns>
+        /// <remarks>Assumes that the attribute name is stored in the first child of the metric row.</remarks>
+        private static TextMeshProUGUI Attribute(GameObject metricRow)
+        {
+            return metricRow.transform.GetChild(0).GetComponent<TextMeshProUGUI>();
+        }
+
+        /// <summary>
+        /// Returns the TMP container of <paramref name="metricRow"/> holding the attribute value.
+        /// Parameter <paramref name="metricRow"/> is assumed to represent a row in the metric window providing
+        /// the name and value of a node attribute (metric).
+        /// </summary>
+        /// <param name="metricRow">a game object representing a pair of an attribute name and an attribute value</param>
+        /// <returns>the TMP holding the attribute value</returns>
+        /// <remarks>Assumes that the attribute name is stored in the second child of the metric row.</remarks>
+        private static TextMeshProUGUI Value(GameObject metricRow)
+        {
+            return metricRow.transform.GetChild(1).GetComponent<TextMeshProUGUI>();
+        }
+
+        /// <summary>
+        /// The GameObject representing the metric window. It must be stored in this field;
+        /// otherwise, it will be garbage collected.
+        /// </summary>
+        private GameObject metricWindowObject;
```

Comment:
Wait, really? Does this mean that game objects can get automatically destroyed if variables referencing them get garbage collected?

Reply:
No, actually not. Originally it was a field, which I then turned into a local variable. But that failed. So I reverted my change. 

I now turned it back into a local variable again and it worked.

Apparently, the previous error was caused by something else.